mobile/lib/common/settings_screen.dart

```diff
@@ -66,6 +73,19 @@ class SettingsScreen extends StatelessWidget {
             ),
           ],
         ),
+        ElevatedButton(
+            onPressed: () async {
+              var file = await FLog.exportLogs();
+              var logsAsString = await file.readAsString();
+              final List<int> bytes = utf8.encode(logsAsString);
+              final Directory tempDir = await getTemporaryDirectory();
+              String now = DateFormat('yyyy-MM-dd_HH:mm:ss').format(DateTime.now());
+              final String filePath = '${tempDir.path}/$now.logs';
+              await File(filePath).writeAsBytes(bytes);
+              final XFile logFile = XFile(filePath);
```

Comment:
🤔  `Flog.exportLogs()` is already writing the file to the disk, why do read it and write it again? Shouldn't it be suffcient to simply do the following?

```
final file = await.FLog.exportLogs();
find XFile logFile = XFile(file.path);
String now = DateFormat('yyyy-MM-dd_HH:mm:ss').format(DateTime.now());
Share.shareXFiles([logFile], text: 'Logs from $now');
```

Reply:
I tried this but it did not work.